custom/icds_reports/tests/test_aggregation_script.py

```diff
@@ -0,0 +1,212 @@
+import os
```

Comment:
This file was heavily copy/pasted from the other one under `agg_tests`, but now has a few additions. Apologies for not keeping a git history when I was figuring that out